tests/backup/backup_helper.go

```diff
@@ -1477,3 +1478,15 @@ func GetLatestScheduleBackupUID(ctx context.Context, scheduleName string, orgID
 	}
 	return allScheduleBackupUids[len(allScheduleBackupUids)-1], nil
 }
+
+// IsPresent verifies if the given data is present in slice of data
+func IsPresent(listOfData interface{}, data interface{}) bool {
```

Comment:
If the intention is for the user to pass a slice, perhaps it would be clearer to name the parameter "dataSlice" or something similar. Although it does not change anything.

Reply:
done